app/controllers/hub/notes_controller.rb

```diff
@@ -28,7 +28,11 @@ def note_params
 
     def create_notifications
       mentioned_user_ids = params.dig(:note, :mentioned_ids).split(",")
-      mentioned_user_ids.each { |id| UserNotification.create(notifiable: @note, user_id: id) }
+      mentioned_user_ids.each do |id|
+        UserNotification.create(notifiable: @note, user_id: id)
+        mentioned_user = User.find(id)
+        InteractionTrackingService.record_internal_interaction(@note.client, user: mentioned_user, interaction_type: "tagged_in_note", received_at: @note.created_at) if mentioned_user
+      end
```

Comment:
🤔 why would you have a mentioned_user_id but not a mentioned user?

Reply:
yeahh i think this was me being lazy and paranoid. if this ever throws an error it would be good for us to see it instead of silently failing to find a user.